### DISCUSSION TITLE
슉 인터뷰

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#61] [General]
# 슉 인터뷰

인터뷰 주제 주노 하루에 노래를 얼마나 들으세요 ? 3시간 이상 노래를 언제 들으시나요 ?   출퇴근 시간, 개인 공부할 때 유투브로도 음악을 들으시나요 ?   유튜브 뮤직, 유튜브로만 듣는다. 유투브에 있는 플레이리스트들을 이용해 보셨나요?   예…

https://vero.wiki/shook-planning-interview/

**Comment by Choi-JJunho** [comment]
왜 우리팀 사람들 괴롭히나요

**Reply by Cyma-s** [reply]
다들 즐거워했어요 분명
